### PR TITLE
[DOC] enable edit on github

### PR DESCRIPTION
**Pull request by szha**
## Description ##
enable edit on github button

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [x] enable edit on github button

## Comments ##
- the auto-generated API docs will need additional work to function correctly.

### CODE REVIEW

**Comment by ys2843** [comment]
Link: http://ec2-34-219-134-42.us-west-2.compute.amazonaws.com/api/python/docs/tutorials/

**Comment by szha** [comment]
@ys2843 right, I think those are due to the fact that the published files and the sources are not the same. this applies to md->ipynb, generated rst, etc.

File: docs/python_docs/python/scripts/conf.py [hunk 1]
```diff
@@ -237,6 +234,16 @@
 # let the source file format to be xxx.ipynb instead of xxx.ipynb.txt
 html_sourcelink_suffix = ''
 
+html_context = {
+    'display_github': True,
+    'github_user': 'apache',
+    'github_repo': 'mxnet',
```
Comment (marcoabreu) [inline docs/python_docs/python/scripts/conf.py:240]: incubator-mxnet?

Comment (szha) [reply docs/python_docs/python/scripts/conf.py:240]: There is forward in the link and I think the link is future proof 😆

**Comment by ys2843** [comment]
There are still some broken links: http://ec2-34-219-134-42.us-west-2.compute.amazonaws.com/api/python/docs/tutorials/

**Review by ys2843 — APPROVED** [review]
LGTM! Didn't find broken links any more.
Link: http://ec2-34-219-134-42.us-west-2.compute.amazonaws.com/api/python/docs/tutorials/index.html

**Pull request merged**